.semaphore/semaphore.yml

```diff
@@ -1,23 +1,23 @@
-version: v1.0
-name: Build & Test
-agent:
-  machine:
-    type: e1-standard-4
-    os_image: ubuntu1804
-
-blocks:
-  - name: Build & Test
-    task:
-      env_vars:
-        # Set maven to use a local directory. This is required for
-        # the cache util. It must be set in all blocks.
-        - name: MAVEN_OPTS
-          value: "-Dmaven.repo.local=.m2"
-      jobs:
-        - name: Build & Test
-          commands:
-            - sem-version java 11
-            - checkout
-            - cache restore maven
-            - mvn scoverage:report
-            - cache store maven .m2
+version: v1.0
+name: Build & Test
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+
+blocks:
+  - name: Build & Test
+    task:
+      env_vars:
+        # Set maven to use a local directory. This is required for
+        # the cache util. It must be set in all blocks.
+        - name: MAVEN_OPTS
+          value: "-Dmaven.repo.local=.m2"
+      jobs:
+        - name: Build & Test
+          commands:
+            - sem-version java 11
+            - checkout
+            - cache restore maven
+            - mvn scoverage:report && bash <(curl -s https://codecov.io/bash)
```

Comment:
Is it possible to put this on the next line?
```
- mvn scoverage:report
- bash <(curl -s https://codecov.io/bash)
```

Reply:
I already tried it in 7756c5e but it doesn't work (codecov upload was executed but the report wasn't pushed on the PR)